public/app/plugins/panel/dashlist/DashList.tsx

```diff
@@ -240,7 +240,7 @@ function IconToggle({
   const iconPropsOverride = checked ? enabled : disabled;
   const iconProps = { ...otherProps, ...iconPropsOverride };
   const styles = useStyles2(getCheckboxStyles);
-  return (
+  const toggle = (
     <label className={styles.wrapper}>
```

Comment:
Because of the interaction of the tooltip cloning its immediate child (...and adding `tabIndex=0`) I'm finding this label is causing issues when I'm navigating with my keyboard. You stop once on the label with the tooltip, which you think is the interactive element (but doesn't do anything), then tab again to get to the checkbox.

I think if we change the implementation to a button (potentially utilizing `aria-pressed`... but maybe not - see other comment) that will address this issue. I don't think I can see any adverse affects from going input/type=checkbox to button/aria-pressed.

What do you think?